### ISSUE TITLE
✅ 🐞 Bug Report: Fullscreen Toolbar blocked by Elementor UI - [ED-11865]

### ISSUE THREAD

**Issue by JoelEadeDesign** [elementor/elementor#22873]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

When editing in the Text Editor widget, if you click on the Fullscreen icon the Toolbar icons are hidden behind the Elementor UI. 

![image](https://github.com/elementor/elementor/assets/1729832/2c97a39f-ce00-4fa1-97d5-4815d1ca242b)

Here's a video of me hiding the Elementor UI (disabling position: sticky) which reveals the Text Editor toolbar in Fullscreen view: https://www.loom.com/share/49ecb00394e247619b70a9bf0e697d09?sid=1bd20f47-4057-4c5e-8ed1-fc1301b7af1f

### Steps to reproduce

1. Add a Text Editor widget to a page
2. Click the Fullscreen icon

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v10.6.14
	PHP Version: 7.4.33
	PHP Memory Limit: 256M
	PHP Max Input Vars: 3000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2.2
	Site URL: http://110.232.143.4/~signumpr
	Home URL: http://110.232.143.4/~signumpr
	WP Multisite: No
	Max Upload Size: 128 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en
	Timezone: 0
	Admin Email: [REDACTED]
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36

== Active Plugins ==
	Advanced Custom Fields
		Version: 6.1.6
		Author: WP Engine

	Elementor
		Version: 3.14.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.14.0
		Author: Elementor.com

	Jetpack Boost
		Version: 1.9.1
		Author: Automattic - Jetpack Site Speed team

	Limit Login Attempts Reloaded
		Version: 2.25.20
		Author: Limit Login Attempts Reloaded

	MantisProperty
		Version: 3.0
		Author: Mantis Technologies

	Tuxedo CSS Editor
		Version: 1.1
		Author: Trevor Anderson

	Widgets for Google Reviews
		Version: 10.2.1
		Author: Trustindex.io <support@trustindex.io>

	WP Super Cache
		Version: 1.9.4
		Author: Automattic


== Must-Use Plugins ==
	WordPress automation by Installatron
		Version: 
		Author: 


== Elements Usage ==
	
	container : 1
		container : 1
		heading : 1
	footer : 1
		button : 1
		container : 6
		heading : 7
		image : 1
		nav-menu : 1
		social-icons : 1
		spacer : 3
		text-editor : 1
	header : 2
		button : 2
		container : 2
		nav-menu : 1
		theme-site-logo : 1
	popup : 1
		container : 3
		form : 1
		theme-site-logo : 1
	widget : 0
		form : 1
	wp-page : 7
		button : 2
		container : 24
		form : 1
		google_maps : 1
		heading : 20
		loop-carousel : 1
		shortcode : 4
		social-icons : 1
		spacer : 2
		text-editor : 4
	loop-item : 1
		container : 3
		heading : 2
		icon-list : 1
		theme-post-content : 1
		theme-post-featured-image : 1
		theme-post-title : 1


== Settings ==
	
	cpt_support: page
	disable_color_schemes: yes
	disable_typography_schemes: yes
	allow_tracking: yes
	unfiltered_files_upload: 1
	font_display: swap


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimized DOM Output: Active
	Improved Asset Loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Inactive
	Accessibility Improvements: Active
	Additional Custom Breakpoints: Inactive
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Grid Container: Inactive
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Active by default
	Editor Top Bar: Active
	Landing Pages: Inactive
	Nested Elements: Active
	Nested Accordion: Inactive by default
	Lazy Load Background Images: Active
	Global Style Guide: Active
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Inactive by default


== Log ==
	
PHP: showing 5 of 5PHP: 2023-06-23 01:01:49 [notice X 6][/home/signumpr/public_html/wp-content/plugins/elementor/app/modules/import-export/module.php::734] Trying to get property 'label' of non-object [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-06-23 01:01:50 [notice X 7][/home/signumpr/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-06-23 03:20:03 [notice X 2][/home/signumpr/public_html/wp-content/plugins/elementor/modules/history/revisions-manager.php::363] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-06-23 03:22:03 [notice X 3][/home/signumpr/public_html/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined index: condition_type [array (
  'trace' => '
#0: /home/signumpr/public_html/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: /home/signumpr/public_html/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: /home/signumpr/public_html/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: /home/signumpr/public_html/wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2023-06-23 03:23:03 [notice X 2][/home/signumpr/public_html/wp-content/plugins/elementor-pro/modules/dynamic-tags/acf/tags/acf-text.php::33] Undefined offset: 1 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 5 of 52023-06-23 01:04:16 [info] elementor-pro::elementor_pro_updater Started 
2023-06-23 01:04:16 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-06-23 01:04:16 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-06-23 01:04:16 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.10.1',
  'to' => '3.14.0',
)]
2023-06-23 01:04:16 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.10.1',
  'to' => '3.14.0',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
	

				

````

</details>

**Comment by nicholaszein** [comment]
Hello @JoelEadeDesign! 

Thank you for submitting your issue! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by nicholaszein** [comment]
Hi, @JoelEadeDesign,

Thank you for reaching out! 🙏 

### ⛔ After reviewing your report we concluded this is not a bug caused by Elementor. The feature you're referring to is not yet supported.

If you think this should be an Elementor feature, please submit a new [🚀 Feature Request](https://github.com/orgs/elementor/discussions/new?category=feature-request), and we will consider it in the future if there is enough demand (votes) from other users.

Kind regards

**Comment by nicholaszein** [comment]
Hey @JoelEadeDesign!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor v3.17! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
